### ISSUE TITLE
NG Input: Preferences/Shortcuts no longer has a search facility

### ISSUE THREAD

**Issue by elstoc** [dterrahe/darktable#12]
This needs to be reinstated.

Ref: darktable-org#8078

**Comment by dterrahe** [comment]
How do you mean "reinstated"? Its a completely new dialog 😄 

But... it _does_ have a search facility. Its implemented via the "standard" gtk incremental search functionality. Make sure the desired list is active and start typing. It will select the highest level of the tree that contains the string; to get to a particular instance, press the down and up keys.

This is the same functionality that is partially implemented in the presets tab of the preference dialog. Except there, it only seems to work if you type the _start_ of the _preset_ name (it doesn't search module names; fair enough, they are easy to find at top level) but it only finds presets for modules that are expanded. In the shortcuts dialog it will expand the tree if needed to show the matching items (but it will first select the top level containing it, until you press the down button to cycle, which can be useful to know where you are)

**Comment by elstoc** [comment]
Yes this is why I wrote a proper search for the shortcuts dialog - because the gtk one sucks.

**Comment by dterrahe** [comment]
But mine is a lot better than the presets one...
